src/components/modals/ReferenceModal.vue

```diff
@@ -56,6 +56,9 @@ export default {
   components: {
     Modal
   },
+  mounted: function() {
+    this.$children[0].isMaximized = true;
```

Comment:
Please don't make it maximized by default. It's okay to make it remember the maximize state for future use, but keep it "minimized" by default to match the other modals.

Reply:
if that's wanted, ok